docs/src/content/docs/EditLiveUpdate.md

```diff
@@ -0,0 +1,108 @@
+---
+title: EditLiveUpdate
+---
+
+`<EditLiveUpdate>` is a component that warns user when a record has been updated in an [`<Edit>`](./Edit.md) view when a record is updated.
+
+This feature requires a valid [Enterprise Edition](https://marmelab.com/ra-enterprise/) subscription.
+
+![A notification alerting users that the record has changed inside an edit view](./images/edit-live-update.png)
+
+## Creating the Component
```

Comment:
since that's not a ready-made component, I don't think we should include a standalone file to document it. Instead, you should move these instructions inside the Edit.md file.